### PR TITLE
Remove leveldown, stip absolute paths from test mk 2, accept reordered and new user baselines

### PR DESCRIPTION
**Pull request by weswigham**
This is all our recent user test changes, along with a small patch to remove the absolute path from the baseline, and the reordering I still hadn't accepted baselines for.

### CODE REVIEW

File: tests/baselines/reference/user/chrome-devtools-frontend.log [hunk 1]
```diff
@@ -6666,10 +6666,14 @@ node_modules/chrome-devtools-frontend/front_end/externs.js(79,17): error TS2551:
 node_modules/chrome-devtools-frontend/front_end/externs.js(82,13): error TS2355: A function whose declared type is neither 'void' nor 'any' must return a value.
 node_modules/chrome-devtools-frontend/front_end/externs.js(86,17): error TS2339: Property 'rotate' does not exist on type 'any[]'.
 node_modules/chrome-devtools-frontend/front_end/externs.js(90,17): error TS2339: Property 'sortNumbers' does not exist on type 'any[]'.
+node_modules/chrome-devtools-frontend/front_end/externs.js(92,13): error TS2304: Cannot find name 'S'.
```
Comment (sandersn) [inline tests/baselines/reference/user/chrome-devtools-frontend.log]: That's interesting. These are new? Oh wait. Does this include changes from chrome-devtools-frontend, or is it pinned?

File: tests/baselines/reference/user/chrome-devtools-frontend.log [hunk 2]
```diff
@@ -9232,6 +9236,13 @@ node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1164,15):
 node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1169,24): error TS2304: Cannot find name 'KEY'.
 node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1169,30): error TS2304: Cannot find name 'VALUE'.
 node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1171,15): error TS2339: Property 'inverse' does not exist on type 'Map<any, any>'.
+node_modules/chrome-devtools-frontend/front_end/platform/utilities.js(1191,14): error TS2304: Cannot find name 'K'.
```
Comment (sandersn) [inline tests/baselines/reference/user/chrome-devtools-frontend.log]: These are new too.

File: tests/baselines/reference/user/chrome-devtools-frontend.log (hunk 1 above)
Comment (weswigham) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: Pretty sure it's pinned.

Comment (weswigham) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: image
Yep, pinned.

Comment (weswigham) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: This is the location of the error, ish:
image

Comment (weswigham) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: Looks like @template on generic methods is borked.

File: tests/baselines/reference/user/chrome-devtools-frontend.log (hunk 2 above)
Comment (weswigham) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: image
Also templates - this time templates for prototype methods not having the templates declared for the constructor (though they should).

Comment (sandersn) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: Yep, looks like the check I put in resolveEntityName needs to understand prototype assignments in addition to the prototype method assignments it already does.

Comment (sandersn) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: #24230

File: tests/baselines/reference/user/chrome-devtools-frontend.log (hunk 1 above)
Comment (sandersn) [reply tests/baselines/reference/user/chrome-devtools-frontend.log]: #24232

**Pull request merged**